llvm/include/llvm/Object/ELF.h

```diff
@@ -321,6 +321,11 @@ class ELFFile {
 
   std::vector<Elf_Rel> decode_relrs(Elf_Relr_Range relrs) const;
 
+  uint64_t crelHeader(ArrayRef<uint8_t> Content) const;
```

Comment:
It's not clear without looking at the method definition what the return value actually represents. Indeed, this method name doesn't seem to conform to the LLVM naming guidelines, since it's neither a verb or adjective phrase or similar. Perhaps renaming the method would be wise (and also possibly a comment to explain it, if needed still after the renaming).

Reply:
renamed to `getCrelHeader`